users/README.md

```diff
@@ -216,6 +215,61 @@ Returns badges assigned to the user
     - **Content:**
       `{ 'statusCode': 400, 'error': 'Bad Request', 'message': 'Failed to get user badges.' }`
 
+## **GET /users/search**
+
+Returns users based on the specified filters.
+
+- **Params:**  
+  None
+
+- **Query Parameters:**
+  - Optional: `levelId=[string]` (Specifies the level ID)
+  - Optional: `levelName=[string]` (Specifies the level name)
+  - Optional: `levelValue=[number]` (Specifies the level value)
+  - Optional: `tagId=[string]` (Specifies the tag ID)
+  - Optional: `state=[string]` (Specifies the user state, can be repeated for multiple states)
+  - Optional: `role=[string]` (Specifies the user role, valid values are "MEMBER", "INDISCORD", "ARCHIVED")
+  - Optional: `verified=[string]` (Specifies if the user is verified)
```

Comment:
NIT: if you can add the value of it here then, it will be helpful

Reply:
Added